### PR TITLE
Update change-address-contact-and-more.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed steps for Changing Alternate email address.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3537

File: microsoft-365/admin/manage/change-address-contact-and-more.md [hunk 1]
```diff
@@ -69,14 +69,15 @@ As an admin, you can change your alternate email address for resetting passwords
...
 1. Go to the <a href="https://go.microsoft.com/fwlink/p/?linkid=2024339" target="_blank">Microsoft 365 admin center</a>.
 
-2. In the header, select your profile icon, then select **My account**.
+2. In the header, select your profile icon, then select **View account**.
 
-3. On the left, select **Personal info**.
+3. On the left, select **Security info**.
 
-4. On the **Contact details** card, select **Edit**.
+4. In the Eamail section, Click on **Change**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/change-address-contact-and-more.md]:
```suggestion
4. In the Email section, select **Change**
```

File: microsoft-365/admin/manage/change-address-contact-and-more.md [hunk 2]
```diff
@@ -69,14 +69,15 @@ As an admin, you can change your alternate email address for resetting passwords
...
 
-5. Edit the address in the **Alternate email** box, then select **Save**.
+5. Edit the address in the **Alternate email** box, then select **Next**.
 
+6. Enter the Code  sent to the Alternate email and Click on **OK** to verify the change.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/change-address-contact-and-more.md]:
```suggestion
6. Enter the code sent to the alternate email address and select **OK** to verify the change.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Pull request merged**